server/csp.ts

```diff
@@ -0,0 +1,61 @@
+// CSP eller Content-Security-Policy er en HTTP-Header som lar oss spesifisere hvor appen kan kjøre REST-kall mot og hvor den kan hente diverse innhold fra (fonter, bilder, javascript, stylesheets mm).
+export const cspMap = (): Record<string, string[]> => {
```

Comment:
Vet ikke om det er relatert, men vi bør kanskje sjekke: 

Failed to load resource: net::ERR_CONNECTION_REFUSED på [www.intern.dev.nav.no/tms-varsel-api/varselbjelle/varsler:1](http://www.intern.dev.nav.no/tms-varsel-api/varselbjelle/varsler:1)
Realtert til csp?

Reply:
Dette ser ikke ut til å være relatert til CSP, men en 503 ?

Reply:
Jeg får det i alle fall. Skal sjekke litt